documentation/database/mysql.md

```diff
@@ -29,6 +29,11 @@ subtitle: MySQL
 <td><code>jdbc:mysql://<i>host</i>:<i>port</i>/<i>database</i></code></td>
 </tr>
 <tr>
+<th>SSL support</th>
+<td>Yes - add <code>?useSsl=true</code></td>
+<td>Yes - add <code>?useSsl=true</code></td>
```

Comment:
Duplicated line

Reply:
This is intentional, it's putting two columns into a row with the same value in each